homeassistant/components/airnow/sensor.py

```diff
@@ -94,8 +94,8 @@ class AirNowEntityDescription(SensorEntityDescription, AirNowEntityDescriptionMi
         icon="mdi:blur",
         value_fn=lambda data: data.get(ATTR_API_STATION),
         extra_state_attributes_fn=lambda data: {
-            "lat": data[ATTR_API_STATION_LATITUDE],
-            "long": data[ATTR_API_STATION_LONGITUDE],
+            "lat": data.get(ATTR_API_STATION_LATITUDE),
+            "long": data.get(ATTR_API_STATION_LONGITUDE),
         },
```

Comment:
Do we want to return nothing or {lat: None, long: None}?

Reply:
You mean returning an empty dict if the `ATTR_API_STATION_*` keys aren't set?
```py
extra_state_attributes_fn=lambda data: {} if ATTR_API_STATION_LATITUDE not in data else {
    "lat": data.get(ATTR_API_STATION_LATITUDE),
    "long": data.get(ATTR_API_STATION_LONGITUDE),
},
```
That's fine with me - whatever is going to be consistent with other integrations.

Reply:
I think omitting the attribute is what is commonly seen, but I don't remember seeing this written down anywhere -- i think its a good suggestion.

Reply:
I would put it in an outside function tho to avoid using multi line ternaries (or how you call it)